pages/Home.vue

```diff
@@ -60,8 +60,6 @@ import { mapState, mapGetters } from 'vuex';
 import LazyHydrate from 'vue-lazy-hydration';
 import { Logger } from '@vue-storefront/core/lib/logger';
 import Home from '@vue-storefront/core/pages/Home';
-import { registerModule } from '@vue-storefront/core/lib/modules';
-import { RecentlyViewedModule } from '@vue-storefront/core/modules/recently-viewed';
```

Comment:
There is no "recently viewed" section/page/component in Capybara and Storefront UI designs.